### PR TITLE
[JBIDE-21325] allow to disable password prompt when in conn dialog

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

**Comment by adietish** [comment]
@fbricon plz review

File: plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizardPageModel.java [hunk 1]
```diff
@@ -380,6 +382,9 @@ public void setNotConnected() {
 	
 	private void setConnectedStatus(IStatus status) {
 		firePropertyChange(PROPERTY_CONNECTED_STATUS, this.connectedStatus, this.connectedStatus = status);
+		if (isConnected()) {
+			this.connection.enablePromptCredentials(true);
```
Comment (fbricon) [inline plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizardPageModel.java]: can the connection be null here?

Comment (fbricon) [inline plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizardPageModel.java]: probably not if isConnected returns true

**Comment by fbricon** [comment]
tested it works.
+1

**Comment by adietish** [comment]
great, thx!

**Comment by adietish** [comment]
merged and pushed to upstream/jbosstools-4.3.1.Beta1x

**Pull request closed without merging**